lib/rialto/etl/service_client/retriable_connection_factory.rb

```diff
@@ -25,7 +25,7 @@ def self.build(uri:, headers:, logger: default_logger)
                                        retry_statuses: retry_statuses
 
             connection.ssl.update(verify: true, verify_mode: OpenSSL::SSL::VERIFY_PEER)
-            connection.adapter :net_http_persistent
+            connection.adapter :net_http
```

Comment:
Can you add a comment as to why using net_http instead of net_http_persistent?

Reply:
Yep! Done.